src/main/java/com/vmware/vipclient/i18n/base/cache/Cache.java

```diff
@@ -4,27 +4,36 @@
  */
 package com.vmware.vipclient.i18n.base.cache;
```

Comment:
The old cache expiration mechanism conflicts with it in the new design. When the CacheExpiredTime is set, it will clear all the cache when it's time.

Reply:
It doesn't conflict. It is just different. So we allow both:
1. If CacheExpiredTime is set in vipconfig.properties, then the client library will use the "old" caching like you said, it will clear the entire cache when it is expired. It will not use the max-age from the response.
2. If CacheExpiredTime is not set, then the client library will use the "new" caching and use max-age if it is in the http response. If the http response does not have max-age, then the default 864000000 of MessageCache.expiredTime will be used.

There is no conflict. The product application will use either 1 or 2. The product application will not suddenly break when client library is upgraded.